File: Implementation/Algorithms/java/SelectionSort.java
import java.util.*;

public class SelectionSort {
    public static void selectionSort(int[] arr) {
        int ma = Integer.MAX_VALUE, pos = 0;
        for (int i = 0; i < arr.length; ++i) {
            for (int j = i; j < arr.length; ++j) {
                if (arr[j] < ma) {
                    ma = arr[j];
                    pos = j;
                }
            }

            int temp = arr[i];
            arr[i] = arr[pos];
            arr[pos] = temp;
            ma = Integer.MAX_VALUE;
        }
    }

    public static void main(String[] args) {
        int[] arr = {12, 11, 13, 5, 6};
        
        selectionSort(arr);

        for (int i = 0; i < arr.length; ++i) {
            System.out.println(arr[i]);
        }
    }
}

File: Implementation/Algorithms/python/SelectionSort.py
import sys

def selectionSort(arr):
    ma = sys.maxsize
    pos = 0

    for i in range(len(arr)):
        for j in range(i, len(arr)):
            if arr[j] < ma:
                ma = arr[j]
                pos = j

        arr[pos], arr[i] = arr[i], arr[pos]
        ma = sys.maxsize

arr = [12, 11, 13, 5, 6]
selectionSort(arr)

for i in arr:
    print(i, end = " ")
